server/src/main/java/eu/fasten/server/plugins/kafka/FastenKafkaPlugin.java

```diff
@@ -101,7 +100,7 @@ public FastenKafkaPlugin(boolean enableKafka, Properties consumerProperties, Pro
         if (enableKafka) {
             this.connNorm = new KafkaConsumer<>(consumerProperties);
             // For priority connection, the client should be different from the normal one
-            this.connPrio = new KafkaConsumer<>(cloneKafkaPropWithNewClientId(consumerProperties, "_priority"));
+            this.connPrio = new KafkaConsumer<>(KafkaConnector.cloneKafkaConsumerPropWithNewClientId(consumerProperties, "_priority"));
```

Comment:
Why has this been extracted to another class now? I think the original idea was to factor-out the property creation and instantiate both property objects together in the `FastenServer` (or was it `Fasten KafkaPlugin`?), instead of passing one around and creating the other one somewhere else..

Reply:
Done!